src/checkstatus/get_automation_execution_status.py

```diff
@@ -0,0 +1,21 @@
+import os
+import boto3
+import constants
+import utility as util
+import custom_exceptions
+import SSMAutomationExecutionStatusWaiter
+
+def lambda_get_automation_execution_status(event, context):
+    """Method to obtain status of a SSM automation execution"""
+    region = os.environ['Region']
+    ssm = boto3.client('ssm', region)
+    result = {}
+    execution_id = event['output']['automation_execution_id']
+    try:
+        max_attempts = util.get_waiter_max_attempts(context)
```

Comment:
can this be made an environment variable ?
```
max_attempts = int(os.environ.get('MAX_ATTEMPTS', 10))
```

Reply:
its a calculated value as per lambda context https://github.com/intuit/Trapheus/blob/master/src/common/python/utility.py#L74